# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: third_party/rust/spirv_headers/lib.rs
# Commit: ef5dc3e04e5f
# Full Hash: ef5dc3e04e5f271eea0636ab3a495e95cc912f1d
# Author: Dzmitry Malyshau <dmalyshau@mozilla.com>
# Date: 2021-09-04 09:40:24
# Regressor Bug: 1726626
# File Overlap Count: 3
# Description:
#   Bug 1726626 - Move gfx/wgpu into a 3rd party dependency r=jgilbert,bholley
#   
#   This update makes wgpu a vendored dependency instead of having it in gfx/wgpu.
#   
#   ## Notes
# ==============================================================================

diff --git a/third_party/rust/spirv_headers/lib.rs b/third_party/rust/spirv/lib.rs
rename from third_party/rust/spirv_headers/lib.rs
rename to third_party/rust/spirv/lib.rs
--- a/third_party/rust/spirv_headers/lib.rs
+++ b/third_party/rust/spirv/lib.rs
@@ -1,16 +1,14 @@
-#![doc(html_root_url = "https://docs.rs/spirv_headers/1.5/")]
-
-//! The SPIR-V header.
-//!
-//! This crate contains Rust definitions of all SPIR-V structs, enums,
-//! and constants.
-//!
-//! The version of this crate is the version of SPIR-V it contains.
-
-#![allow(non_camel_case_types)]
-#![cfg_attr(rustfmt, rustfmt_skip)]
-
-use bitflags::bitflags;
-use num_traits;
-
-include!("autogen_spirv.rs");
+//! The SPIR-V header.
+//!
+//! This crate contains Rust definitions of all SPIR-V structs, enums,
+//! and constants.
+//!
+//! The version of this crate is the version of SPIR-V it contains.
+
+#![no_std]
+#![allow(non_camel_case_types)]
+#![cfg_attr(rustfmt, rustfmt_skip)]
+
+use bitflags::bitflags;
+
+include!("autogen_spirv.rs");